src/drt/src/db/obj/frInst.h

```diff
@@ -94,6 +95,15 @@ class frInst : public frRef
   void setOrigin(const Point& tmpPoint) override { xform_.setOffset(tmpPoint); }
   dbTransform getTransform() const override { return xform_; }
   void setTransform(const dbTransform& xformIn) override { xform_ = xformIn; }
+  void setDBInst(odb::dbInst* db_inst_in) { db_inst_ = db_inst_in; }
+  odb::dbInst* getDBInst() const { return db_inst_; }
+  dbTransform getDBTransform() const
+  {
+    if (!db_inst_) {
+      return dbTransform();
+    }
```

Comment:
unneeded if previous comment is addressed

Reply:
This exists only as a patch for the gc_test unit test. I was having a lot of trouble with it, as the odb Transform held trash when called. The old version worked because the drt Transform created a default Transform upon initialization 0,0,R0. This piece of code is to replicate the behavior as I thought I was loosing too much time trying to fix the unit test trying to make it work correctly. I will comply with your previous feedback, if it fixes the problem I will delete this piece of code.